restapi/src/main/java/com/craftsmanship/tfm/restapi/config/PersistenceConfig.java

```diff
@@ -24,7 +25,12 @@
   private int serverPort;
 
   @Bean
-  public ItemPersistenceGrpc persistenceHandler() {
+  public ItemPersistenceGrpc itemPersistenceHandler() {
```

Comment:
Would not be better ItemPersistence ?

Reply:
I wanted to let clear that in production ItemPersistenceGrpc will be used. But I can change if you want.